core/test/testmodelarraydetails/ModelArrayDetails.cpp

```diff
@@ -43,6 +49,7 @@ const std::map<ModelArray::Type, std::string> ModelArray::typeNames = {
     { ModelArray::Type::ONED, "OneDField" },
     { ModelArray::Type::TWOD, "TwoDField" },
     { ModelArray::Type::DOSD, "TwoDField" },
+    { ModelArray::Type::DOSD, "ThreeDField" },
```

Comment:
I think this should be `ModelArray::Type::THREED`

As a side note, do we explain anywhere what `DOSD` is? how does it differ from TWOD. I am guessing `DOS` is the number 2 in Spanish?

Reply:
> I think this should be ModelArray::Type::THREED

Indeed it should.

DOSD has now been renamed to the more descriptive ZUFIELD.